docs/developer-docs/latest/development/admin-customization.md

```diff
@@ -229,16 +229,12 @@ To disable notifications about new Strapi releases, set the `config.notification
 
 #### Theme extension
 
-<!-- TODO: complete this section once design system is ready -->
-
 To extend the theme, use the `config.theme` key.
 
-::: Strapi Design System
-The default [Strapi theme](https://github.com/strapi/design-system/tree/main/packages/strapi-design-system/src/themes) defines various theme-related keys (shadows, colors…) that can be updated through the `config.theme` key in `./admin/src/app.js`.
+::: strapi Strapi Design System
+The default [Strapi theme](https://github.com/strapi/design-system/tree/main/packages/strapi-design-system/src/themes) defines various theme-related keys (shadows, colors…) that can be updated through the `config.theme` key in `./admin/src/app.js`. The [Strapi Design System](https://design-system.strapi.io/) is fully customizable.
```

Comment:
Not sure to understand what to you mean by fully customizable?

Reply:
I'd say "ask @soupette " 😂  This is what he actually wrote. I assumed he meant we can change colors, shadows, etc.

Reply:
Since you can change every part of theme it's fully customizable .